pennylane/gradients/parameter_shift_hessian.py

```diff
@@ -650,16 +650,6 @@ def param_shift_hessian(tape, argnum=None, diagonal_shifts=None, off_diagonal_sh
             "chosen auto differentiation framework, or via the 'tape.trainable_params' property."
         )
         return [], lambda _: qml.math.zeros((tape.output_dim, 0, 0))
-
-    # For Jax the argnum defines the tape trainable parameters (it does not work for an array argnum).
-    if (
-        argnum is not None
-        and not isinstance(argnum, np.ndarray)
-        and not tape.trainable_params
-        and qml.math.get_interface(*tape.data) in ["jax", "jax-jit"]
-    ):
-        tape.trainable_params = argnum
-
```

Comment:
There is a second place in this file where a parallel deletion should be made, no?
See [here](https://github.com/PennyLaneAI/pennylane/blob/a92677ffd977f0fd622f4bedd59095534e8c64fb/pennylane/gradients/parameter_shift_hessian.py#L880)

Reply:
Thanks good catch :+1: